core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -104,7 +104,9 @@ object Partition extends KafkaMetricsGroup {
       metadataCache = replicaManager.metadataCache,
       logManager = replicaManager.logManager,
       alterIsrManager = replicaManager.alterIsrManager,
-      transferLeaderManager = replicaManager.transferLeaderManager)
+      transferLeaderManager = replicaManager.transferLeaderManager,
+      logSlowReplicationThresholdMs = replicaManager.config.longTailProduceRequestLogThresholdMs
+      )
```

Comment:
nit: style here is to keep it on last parameter (L108)

Reply:
Hi Joseph, the new logSlowReplicationThresholdMs is already the last parameter of this function. Is that what you are saying?